src/app/rpmostree-builtin-status.c

```diff
@@ -694,9 +694,9 @@ print_one_deployment (RPMOSTreeSysroot *sysroot_proxy,
 
   /* print base overrides before overlays */
   g_autoptr(GPtrArray) active_removals = g_ptr_array_new_with_free_func (g_free);
+  g_autoptr(GPtrArray) active_removals_nevra = g_ptr_array_new_with_free_func (g_free);
```

Comment:
Minor: we can move this to inside the if statement since we don't need it later on (as opposed to `active_removals`).